src/plugins/expression_shape/public/components/progress/progress_component.tsx

```diff
@@ -113,19 +110,17 @@ function ProgressComponent({
 
   return (
     <div className="shapeAligner">
-      <ProgressDrawer
-        shapeType={shapeType}
-        shapeContentAttributes={{ ...shapeContentAttributes, ref: progressRef }}
-        shapeAttributes={shapeAttributes}
-        textAttributes={{ ...textAttributes, ref: textRef }}
-        ref={shapeRef}
-      >
-        {BarProgress && <BarProgress {...barProgressAttributes} />}
-      </ProgressDrawer>
+      <EuiErrorBoundary>
```

Comment:
error boundary not needed here. Error boundary is added in src/plugins/expression_shape/public/expression_renderers/progress_renderer.tsx where async chain is loaded